cmd/params/params_test.go

```diff
@@ -925,6 +925,18 @@ func TestLoadHeartbeatParams_Filter_ExcludeUnknownProject_FromConfig(t *testing.
 	assert.True(t, params.Filter.ExcludeUnknownProject)
 }
 
+func TestLoadHeartbeatParams_Filter_ExcludeUnknownProject_FlagTakesPrecedence(t *testing.T) {
+	v := viper.New()
+	v.Set("entity", "/path/to/file")
+	v.Set("exclude-unknown-project", false)
+	v.Set("settings.exclude_unknown_project", true)
+
+	params, err := cmdparams.LoadHeartbeatParams(context.Background(), v)
+	require.NoError(t, err)
+
+	assert.False(t, params.Filter.ExcludeUnknownProject)
```

Comment:
It should be falsy despite it's set in config file as flag always takes precedence.